public/app/features/alerting/unified/components/rule-editor/NotificationsStep.tsx

```diff
@@ -1,112 +1,263 @@
+import { css } from '@emotion/css';
 import React from 'react';
 import { useFormContext } from 'react-hook-form';
 
-import { Icon, Text, Stack } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { Icon, RadioButtonGroup, Stack, Text, useStyles2 } from '@grafana/ui';
 
 import { RuleFormType, RuleFormValues } from '../../types/rule-form';
 import { GRAFANA_RULES_SOURCE_NAME } from '../../utils/datasource';
 
 import LabelsField from './LabelsField';
 import { NeedHelpInfo } from './NeedHelpInfo';
 import { RuleEditorSection } from './RuleEditorSection';
+import { SimplifiedRouting } from './alert-rule-form/simplifiedRouting/SimplifiedRouting';
 import { NotificationPreview } from './notificaton-preview/NotificationPreview';
 
 type NotificationsStepProps = {
   alertUid?: string;
 };
+
+enum RoutingOptions {
+  'notification policy' = 'notification policy',
+  'contact point' = 'contact point',
+}
+
 export const NotificationsStep = ({ alertUid }: NotificationsStepProps) => {
-  const { watch } = useFormContext<RuleFormValues & { location?: string }>();
+  const { watch, setValue } = useFormContext<RuleFormValues & { location?: string }>();
```

Comment:
Unrelated to changes in this PR; but is `location` even used in the form?

Reply:
I think we are not using it anywhere 🤔

Reply:
done!